packages/consumption/test/core/TestUtil.ts

```diff
@@ -462,15 +463,16 @@ export class TestUtil {
         });
     }
 
-    public static async uploadFile(from: AccountController, fileContent: CoreBuffer): Promise<File> {
+    public static async uploadFile(from: AccountController, parameters?: { fileContent?: CoreBuffer; expiredAt?: CoreDate; tags?: string[] }): Promise<File> {
         const params: ISendFileParameters = {
-            buffer: fileContent,
+            buffer: parameters?.fileContent ?? CoreBuffer.from(await fs.promises.readFile(`${__dirname}/../__assets__/test.txt`)),
             title: "aFileName",
-            description: "Dies ist eine Beschreibung",
-            filename: "Test.bin",
+            description: "This is a valid file description",
+            filename: "test.txt",
```

Comment:
if you refactor these properties please refactor them properly (`aDescription`, `aFilename`)

Reply:
I think this will bring up be merge conficts as @britsta has changed this on v7..